include/linux/soundwire/sdw.h

```diff
@@ -283,6 +287,8 @@ struct sdw_dp0_prop {
  * @block_pack_mode: Type of block port mode supported
  * @read_only_wordlength: Read Only wordlength field in DPN_BlockCtrl1 register
  * @port_encoding: Payload Channel Sample encoding schemes supported
+ * @num_lanes: array size of @lane_list
+ * @lane_list: indicates which Lanes can be used by DPn
```

Comment:
@bardliao note that this new property will constrain your multi-lane work. It's not enough to check if all peripherals are connected on a shared lane, we need **in addition** to make sure there's an available port to read/write from that lane.